broker-core/src/test/java/io/zeebe/broker/exporter/stream/ExporterDirectorTest.java

```diff
@@ -327,7 +336,7 @@ public void shouldUpdateLastExportedPositionOnClose() {
   }
 
   @Test
-  public void shouldRemoveExporterFromState() {
+  public void shouldRemoveExporterFromState() throws Exception {
```

Comment:
I added a `zeebeDb.close` to `closeExporterDirector()` because of a bug where the lock file wasn't released which would cause an exception when restarting the exporter director.

Reply:
Ah ok thanks